<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_task_superbasictask</class>
 <widget class="QDialog" name="add_task_superbasictask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add SuperBasicTask</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Task Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Frequency:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>Once</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hourly</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Daily</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Weekly</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fortnight</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Monthly</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Quarterly</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yearly</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Infinite</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Date of Next:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>178</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>&amp;Add Task</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>118</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
